src/test/ui/consts/const-mut-refs/feature-gate-const_mut_refs.stderr

```diff
@@ -0,0 +1,12 @@
+error[E0723]: mutable references in const fn are unstable
+  --> $DIR/feature-gate-const_mut_refs.rs:5:14
+   |
+LL | const fn foo(x: &mut i32) -> i32 {
+   |              ^
+   |
+   = note: for more information, see issue https://github.com/rust-lang/rust/issues/57563
+   = help: add `#![feature(const_fn)]` to the crate attributes to enable
```

Comment:
Hmm... it's quite unfortunate that the wrong feature gate is suggested.

Reply:
yes :( I'd like to fix it

Reply:
I was able to fix it when mutable borrows are done inside constant contexts but not when they are function arguments. This is because that check is done by the const fn qualification and it always suggests the `const_fn` flag.